specification.markdown

```diff
@@ -448,116 +448,191 @@ This table clarifies if a Mandatory EBML Element MUST be written, according to i
+
+name: `DocType`
+
+level: 1
+
+id `0x4282`
+
+minOccurs: 1
+
+maxOccurs: 1
+
+type: String
+
+description: A string that describes and identifies the content of the `EBML Body` that follows this `EBML Header`.
+
+### DocTypeVersion Element
+
+name: `DocTypeVersion`
+
+level: 1
+
+id `0x4287`
+
+minOccurs: 1
+
+maxOccurs: 1
+
+range:  -
+
+default: 1
+
+type: Unsigned Integer
+
+description: The version of DocType interpreter used to create the EBML Document.
+
+### DocTypeReadVersion Element
+
+name: DocTypeReadVersion
+
+level: 1
+
+id `0x4285`
+
+minOccurs: 1
+
+maxOccurs: 1
+
+default: 1
+
+type: Unsigned Integer
+
+description: The minimum `DocType` version an `EBML Reader` has to support to read this `EBML Document`. The value of the `DocTypeReadVersion Element` MUST be less than or equal to the value of the `DocTypeVersion Element`.
 
 ## Global elements (used everywhere in the format)
 
-Name         | CRC-32
-:------------|:------
-Level        | 1
-Global       | Yes
-EBML ID      | `0xBF`
-Mandatory    | No
-Range        | -
-Size         | 4
-Default      | -
-Type         | Binary
-Description  | The CRC-32 Element contains a 32 bit Cyclic Redundancy Check value of all the Element Data of the Parent Element as stored except for the CRC-32 Element itself. When the CRC-32 Element is present, the CRC-32 Element MUST be the first ordered Element within its Parent Element for easier reading. All Top-Level Elements of an EBML Document SHOULD include a CRC-32 Element as a Child Element. The CRC in use is the IEEE-CRC-32 algorithm as used in the [@!ISO.3309.1979] standard and in section 8.1.1.6.2 of [@!ITU.V42.1994], with initial value of `0xFFFFFFFF`. The CRC value MUST be computed on a little endian bitstream and MUST use little endian storage.
-
-Name         | Void
-:------------|:----
-Level        | 0
-Global       | Yes
-EBML ID      | `0xEC`
-Mandatory    | No
-Multiple     | Yes
-Range        | -
-Default      | -
-Type         | Binary
-Description  | Used to void damaged data, to avoid unexpected behaviors when using damaged data. The content is discarded. Also used to reserve space in a sub-element for later use.
+
+name: CRC-32
+
+level: 1
+
+global: true
+
+id: `0xBF`
+
+minOccurs: 0
+
+maxOccurs: 1
+
+size: 4
+
+type: Binary
+
+description: The `CRC-32 Element` contains a 32 bit Cyclic Redundancy Check value of all the `Element Data` of the `Parent Element` as stored except for the `CRC-32 Element` itself. When the `CRC-32 Element` is present, the `CRC-32 Element` MUST be the first ordered `EBML Element` within its `Parent Element` for easier reading. All `Top-Level Elements` of an `EBML Document` SHOULD include a `CRC-32 Element` as a `Child Element`. The CRC in use is the IEEE-CRC-32 algorithm as used in the [@!ISO.3309.1979] standard and in section 8.1.1.6.2 of [@!ITU.V42.1994], with initial value of `0xFFFFFFFF`. The CRC value MUST be computed on a little endian bitstream and MUST use little endian storage.
```

Comment:
updated and squashed into https://github.com/Matroska-Org/ebml-specification/pull/101/commits/229e83b3d18eb5dc6162343fa2f65849ee7ae4d9